google/cloud/forseti/services/server.py

```diff
@@ -479,12 +485,39 @@ def main():
         '--enable_console_log',
         action='store_true',
         help='Print log to console.')
+
     args = vars(parser.parse_args())
 
+    if not args['services']:
+        sys.stderr.write('ERROR: please specify at least one service.\n\n')
+        parser.print_usage()
+        sys.exit(1)
```

Comment:
These exit codes seem to be a "magic" number and probably warrant a comment regarding where they're coming from and how to increment and/or not change them in the future.